TombEngine/Game/control/event.h

```diff
@@ -41,6 +41,9 @@ namespace TEN::Control::Volumes
 		Save,
 		Start,
 		End,
+		BeforeLoad,
```

Comment:
I think you need to additionally extend exposed Lua enumeration (similar to what I did in my last 2 commits in main PR).